src/drivers/fluid_winmidi.c

```diff
@@ -458,6 +467,22 @@ fluid_winmidi_parse_device_name(fluid_winmidi_driver_t *dev, char *dev_name)
     return dev_count;
 }
 
+static void fluid_winmidi_autoconnect_build_name(char *name)
+{
+    int i, j, n = 0;
+    int num = midiInGetNumDevs();
+    if (num < 11) {
```

Comment:
Why are you checking for `num < 11` here?

Reply:
If I recall correctly, this is because MAXPNAMELEN is defined as 32 in the classic windows multimedia interface, so this was to prevent exceeding the available room in the device name. But when using Unicode, the limit may be 8 instead?

Reply:
Ok, no problem. I'll think about a more explicit check here.